governator-core/src/main/java/com/netflix/governator/PropertiesPropertySource.java

```diff
@@ -38,4 +52,28 @@ public String get(String key) {
     public String get(String key, String defaultValue) {
         return props.getProperty(key, defaultValue);
     }
+
+    @Override
+    public void configure(Binder binder) {
+        binder.bind(PropertySource.class).toInstance(this);
+    }
+    
+    @Override
+    public int hashCode() {
+        return getClass().hashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
+
+        throw new RuntimeException("Only one PropertiesModule may be installed");
```

Comment:
IllegalStateException would be clearer; usually a bad idea to throw RuntimeException